.github/workflows/cancel_dupes.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
do you need this set of dashes?

Reply:
yamllint complains without them. Apparently starting a YAML file with three dashes is "start of document".

Ref: https://stackoverflow.com/questions/50788277/why-3-dashes-hyphen-in-yaml-file

Reply:
Here's the yamllint rule: https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.document_start